src/posthog-surveys.ts

```diff
@@ -258,7 +258,15 @@ export class PostHogSurveys {
             return nextQuestionIndex
         }
 
-        console.warn('Falling back to next question index due to unexpected branching type') // eslint-disable-line no-console
```

Comment:
try not to disable eslint rules, here it was warning you not to use console and it was IMHO right 😊